wagtail_wordpress_import/test/tests/test_wordpress_item.py

```diff
@@ -279,7 +276,8 @@ def test_items_dict_4(self):
 class WordpressImporterTestsParsePostMeta(TestCase):
     """
     This tests the post meta items retrieved from an XML file
-    are stored in WordpressItem().post_meta_items
+    are extracted correctly in WordpressItem().coerce_post_meta_items()
```

Comment:
**Comment**: Ah, it looks like it used to be called something more specific. (I guess this was missed when renaming.)

Reply:
I updated this test name to reflect the new function name